javascript/index.js

```diff
@@ -4,37 +4,175 @@
-  
-  getInstruction("mashedPotatoes", 1, (step2) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 2, (step3) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 3, (step4) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 4, (step5) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
-    document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
+// getInstruction("mashedPotatoes", 0, (step1) => {
+//   document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`;
+// }, (error) => console.log(error));
+
+// getInstruction("mashedPotatoes", 1, (step2) => {
+//   document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
+// }, (error) => console.log(error));
+
+// getInstruction("mashedPotatoes", 2, (step3) => {
+//   document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`;
+// }, (error) => console.log(error));
 
+// getInstruction("mashedPotatoes", 3, (step4) => {
+//   document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`;
+// }, (error) => console.log(error));
 
+// getInstruction("mashedPotatoes", 4, (step5) => {
+//   document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
+//   document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
+// }, (error) => console.log(error));
 
 // Iteration 1 - using callbacks
-// ...
+
+getInstruction(
+  "mashedPotatoes",
+  0,
+  (step0) => {
+    document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
+    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step0}</li>`;
+    getInstruction(
+      "mashedPotatoes",
+      1,
+      (step1) => {
+        document.querySelector(
+          "#mashedPotatoes"
+        ).innerHTML += `<li>${step1}</li>`;
+        getInstruction(
+          "mashedPotatoes",
+          2,
+          (step2) => {
+            document.querySelector(
+              "#mashedPotatoes"
+            ).innerHTML += `<li>${step2}</li>`;
+            getInstruction(
+              "mashedPotatoes",
+              3,
+              (step3) => {
+                document.querySelector(
+                  "#mashedPotatoes"
+                ).innerHTML += `<li>${step3}</li>`;
+                getInstruction(
+                  "mashedPotatoes",
+                  4,
+                  (step4) => {
+                    document.querySelector(
+                      "#mashedPotatoes"
+                    ).innerHTML += `<li>${step4}</li>`;
+                    getInstruction(
+                      "mashedPotatoes",
+                      5,
```

Comment:
there is no step 5, you can delete this step.
And for the additional line, you just add it yourself, no need to invoke the function again. so like this:
```
    getInstruction(
                    "mashedPotatoes",
                    4,
                    (step4) => {
                      document.querySelector(
                        "#mashedPotatoes"
                      ).innerHTML += `<li>${step4}</li>`;
                              document.querySelector(
                                "#mashedPotatoes"
                              ).innerHTML += `<li>Mashed potatoes are ready!</li>`;
                            },
                            (error) => console.log(error)
                          );
                        },
                        (error) => console.log(error)
                      );
                    },
                    (error) => console.log(error)
                  );
                },
                (error) => console.log(error)
              );
            },
            (error) => console.log(error)
          );
      
```
you need to delete two error lines now.